samtranslator/feature_toggle/feature_toggle.py

```diff
@@ -12,9 +12,6 @@
     SimpleAccountPercentileDialup,
 )
 
-my_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
@c2tarun I believe you where the original author of Feature Toggle. Why was this developed in this way? If we remove this, are there any downstream side effects? I assume this was done for a reason, just not sure what it was.

Reply:
As discussed offline, this is so that we can reference [this config file](https://github.com/aws/serverless-application-model/blob/develop/tests/feature_toggle/input/feature_toggle_config.json) from [sam-translate](https://github.com/aws/serverless-application-model/blob/7eb358731933152df41c56b2e8ce1916186460e1/bin/sam-translate.py#L102-L104) script for testing.

Reply:
If this is just for the script, I would rather break that than do patching like this in the lib. We shouldn't need FeatureToggle from that script anyways. Maybe I am missing something but I would error on not overriding `sys.path` as it's super dangerous.